tensorboard/webapp/reducer_config.ts

```diff
@@ -12,32 +12,32 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 ==============================================================================*/
-import {InjectionToken} from '@angular/core';
+import {InjectionToken, isDevMode} from '@angular/core';
 import {
   Action,
   ActionReducer,
   ActionReducerMap,
   MetaReducer,
 } from '@ngrx/store';
 
+import {storeLogger} from 'ngrx-store-logger';
+
 export interface State {}
 
-// console.log all actions
-export function logger(reducer: ActionReducer<any>): ActionReducer<any> {
-  return (state, action) => {
-    const result = reducer(state, action);
-    console.groupCollapsed(action.type);
-    console.log('prev state', state);
-    console.log('action', action);
-    console.log('next state', result);
-    console.groupEnd();
-
-    return result;
-  };
+export function logger(reducer: ActionReducer<State>): ActionReducer<State> {
+  // We must call isDevMode() within this function, not at file load time,
+  // because of load ordering issues vs. the config/bootstrap setup.
+
+  // TODO(stephanwlee, davidsoergel): Why is this called twice?
+  console.log(
+    isDevMode() ? 'DEV mode: Logger enabled' : 'PROD mode: Logger disabled'
```

Comment:
nit: `isDevMod()` is called twice here. Can we just call it once and use the return value twice?